### ISSUE TITLE
Cannot login

### ISSUE THREAD

**Issue by jamg26** [ananay/passport-apple#37]
post callback is not going through passport apple strategy.

what happening is
login -> callback -> login

Sample:
https://apiv2-staging.grapl.it/api/sso/apple

**Comment by matthiasprieth** [comment]
Hey!
I'm using `passport-apple` for the first time as well! I'm running into the same problem. I've setup several official passportjs strategies already (facebook, google), but never had those kind of problems.

Right now it looks like the signin itself is fine but it's not calling this callback function at all & I don't have any clue of how to debug this. No errors, no exceptions.....
Did someone else run into those problems?

```
passport.use(new AppleStrategy({
    clientID: "",
    teamID: "",
    callbackURL: "",
    keyID: "",
    privateKeyLocation: ""
}, function(req, accessToken, refreshToken, idToken, profile , cb) {
    // This function does not get called. Is there a way to debug this?
    cb(null, idToken);
}));
```

**Comment by ananay** [comment]
Hey! I would suggest looking at this example repo for reference: https://github.com/ananay/passport-apple-example/blob/master/src/index.js. Try cloning this repo: https://github.com/ananay/passport-apple-example/ and configuring it with your credentials.

If it still doesn't work, then it looks like there's an error with your Sign In With Apple configuration on Apple Developer. I've created a SETUP.md file that goes through step by step how to set stuff up: https://github.com/ananay/apple-auth/blob/master/SETUP.md

I have a live example of all of this deployed at: https://passport-apple.ananay.dev/

Good luck!

**Comment by matthiasprieth** [comment]
Hi @ananay ,
thx for your comment. I tried your example with my credentials & it worked! 

So I was comparing your example with my code & found out that your code has the same "login -> callback -> login" problem when I remove the line `app.use(bodyParser.urlencoded({ extended: true }));`

So I was adding this line to my code `server.use(express.urlencoded({ extended: true }));` with `express` instead of the deprecated `bodyParser` like mentioned [here](https://stackoverflow.com/questions/24330014/bodyparser-is-deprecated-express-4). Afterwards I was able to debug.

As a last step I also had to change to POST  (`server.post('*/auth/apple/callback', fun..`) instead of GET and now it works! 👍 
Thank you!
